sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1981,16 +1981,23 @@ pi_result piTearDown(void *) {
   return PI_SUCCESS;
 }
 
+const char SupportedVersion[] = _PI_ESIMD_PLUGIN_VERSION_STRING;
+
 pi_result piPluginInit(pi_plugin *PluginInit) {
   if (PluginInit == nullptr) {
     return PI_ERROR_INVALID_VALUE;
   }
 
+  static int PiVersionLen = strlen(PluginInit->PiVersion);
```

Comment:
Since we are adding more to the version string, we may want to increase the size of `PiVersion` to make room for the new version. Especially since I can imagine the plugin version increasing fairly frequently.

Reply:
`PiVersion` will just hold the Major and Minor version of the PI. Whereas `PluginVersion` will hold major minor as well as the `_PI_*_PLUGIN_VERSION`. Nevertheless I have increased both to 20